lib/todo/tasks_formatter.ex

```diff
@@ -17,7 +17,7 @@ defmodule ToDo.TasksFormatter do
   end
 
   defp append_to_do_heading(display_tasks) do
-    "To Do List:\n#{display_tasks}"
+    "\nTo Do List:\n#{display_tasks}"
```

Comment:
Having a new line on the beginning of a string is a bit smelly. Why does this need to be here?